### PR TITLE
Update wallet07.mdx

### PR DESCRIPTION
**Pull request by johnnygreeney**
Updated so this page now contains 2 queries as per request on https://cardanofoundation.atlassian.net/browse/LSY-160

Note indentation can appear skewed in github preview. Renders fine on localhost and live with nextra

@Sotatek-QuanLeA @darlisagc @satran004

### CODE REVIEW

File: docs/pages/use-cases/WalletQueries/wallet07.mdx [hunk 1]
```diff
@@ -2,7 +2,108 @@ import { Callout, Code, Tabs, Steps } from "nextra/components"
 
 ## 7. Query Active Stake Pools
 
-[Staking](https://cardano.org/stake-pool-delegation/) is a big part of the Cardano ecosystem. To get started, the first step is to identify the active stake pools. Our next SQL query retrieves information about active Cardano staking pools, retrievign the pool's ID, name and creation date. 
+[Staking](https://cardano.org/stake-pool-delegation/) is a big part of the Cardano ecosystem. The more healthy stake pools in operation, the more secure and resilient the network becomes. 
+
+### 7.1 Query all Stake Pools
+
+To get started, the first step is to identify all stake pools. This SQL query retrieves information about active Cardano staking pools, retrieving the pool's ID, name and creation date.
```
Comment (darlisagc) [inline docs/pages/use-cases/WalletQueries/wallet07.mdx:9]: "This SQL query retrieves information about active Cardano staking pools".
All stake pools also include retired, so can "active" be removed? This is the next query :)

File: docs/pages/use-cases/WalletQueries/wallet07.mdx [hunk 2]
```diff
@@ -65,7 +166,7 @@ Expected results format
 <Steps>
 ### Select Specific Columns
 
-In order to gather information about staking pools, the query focuses on:
+Similar to our first part, we want to retrieve data for the following columns:
```
Comment (darlisagc) [inline docs/pages/use-cases/WalletQueries/wallet07.mdx:169]: Maybe on the explanation we can just mentioned that query is similar to the previous one with the addition of the "Where" clause, which then we can explain?

Comment (johnnygreeney) [reply docs/pages/use-cases/WalletQueries/wallet07.mdx:169]: @darlisagc updated now, please check, thanks

**Comment by johnnygreeney** [comment]
hi @darlisagc @satran004 this PR has been updated. 

- It includes fixes to relative paths (from September update)
- Now also includes update to the 'README' (GitHub) and the 'Overview page' (https://cardano-foundation.github.io/cf-ledger-sync)
- I added the animation as an MP4 file. Viewer must click 'full screen' to see it properly. 

Please review, thanks

**Review by satran004 — APPROVED** [review]
Thanks @johnnygreeney

**Pull request merged**